Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -801,7 +801,7 @@ class BrowserViewController: UIViewController {
                 
                 let content = UNMutableNotificationContent().then {
                     $0.title = Strings.DefaultBrowserCallout.notificationTitle
-                    $0.body = Strings.DefaultBrowserCallout.notificationBody
+                    $0.body = String(format: Strings.DefaultBrowserCallout.notificationBody, String(floor(Double(UIDevice.current.systemVersion) ?? 14)) ?? "14")
```

Comment:
There is a simpler method to get device's version, try calling it with
`ProcessInfo().operatingSystemVersion.majorVersion`, this should get you a nice Int value